### DISCUSSION TITLE
Stop or piston to execute shell command?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#2049] [Help]
Is there a way to assign a specific stop or piston to a shell command? I need it to reduce the number of buttons needed for turning off the VPO.

//Jens

**Comment by oleg68** [comment]
For now it is not possible.

**Reply by jason-barba** [reply]
So, an alternative:
Does any Ubuntu user know of a way to change the 60 second delay at shutdown or restart to another, smaller value? I tried a HOWTO on a VM Ubuntu but that ended up in disaster.
I run Ubuntu 22.04.1 LTS.

**Reply by thbeckpr** [reply]
Well, sudo shutdown now works but it would require a password.

**Reply by jason-barba** [reply]
Yes that's obvious ;-)
But I am looking for ways to decrease the 60 sec delay in the shutdown and restart pop-up menus.

**Reply by kerkovits** [reply]
Is not this close to what you want to achieve?
https://askubuntu.com/questions/1272300/how-to-disable-shutdown-confirmation-on-ubuntu-20-04

**Reply by jason-barba** [reply]
Yes, I tried that on a VM Ubuntu 24.04 LTS. It resulted in black screen after login. Thus unfortunately a negative experience.
Reading the full thread I can allow me to understand that the problem is not easily solved. 
I think I have to accept and go forward with my life.

**Reply by kerkovits** [reply]
I see. Then it should be possible to deny permission from GNOME to handle the shutdown button event. If it succeeds, then the power button event is sent to systemd, and low-level actions can be controlled in /etc/systemd/logind.conf
Tutorial (see option 2 for denying permission for GNOME):
https://medium.com/@buzondefede/shutdown-from-power-button-explained-in-ubuntu-22-04-3fbe671b5765

**Reply by jason-barba** [reply]
I have briefly scanned the article and find it interesting but somewhat demanding. I will contemplate a bit ...
